cmd/restic/cmd_stats.go

```diff
@@ -15,18 +14,19 @@ import (
 )
 
 var cmdStats = &cobra.Command{
-	Use:   "stats [flags] [snapshot-ID]",
+	Use:   "stats [flags] [snapshot ID] [...]",
 	Short: "Scan the repository and show basic statistics",
 	Long: `
 The "stats" command walks one or all snapshots in a repository and
```

Comment:
Shouldn't this line have "one or more snapshots" rather than "one or all snapshots"?